content/posts/2022-08-23-hot-interconnects.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "OPI Project at Hot Interconnects 2022"
+date: 2022-08-23T05:06:30-06:00
+---
+
+At the recent [Hot Interconnects](https://hoti.org) event, a presentation on
+the Open Programmable Infrastructure Project was presented by Kyle Mestery
+(Intel) and Boris Glimcher (Dell). See below for the link to the recording.
+
+{{< youtube id="hVRopgutVSg?start=7056" title="OPI Presentation at Hot Interconnects 2022" >}}
+
+You can view the slides [here](/presentations/Hot_Interconnects_2022_Presentation.pdf)
```

Comment:
Is this the correct relative link for the presentation? Should it be /static/presentations/Hot_Interconnects_2022_Presentation.pdf ?